### DISCUSSION TITLE
Same key/wallet, two different machine with two different ip

### DISCUSSION THREAD

**Discussion by Peppins32** [Chia-Network/chia-blockchain#6551] [Support]
As title say, i have two plotting machines at two different location, so with two different ip, of course i have the same key on both, so the plots are combined (right?).

But for some reason, on one machine i see the wallet with xch, on another machine i have 0 xch, why? The wallet isn't the same with same key? Why the gui don't show the same wallet stats on both machine?

Maybe the gui only work for local stats like number of plots, total size, time to win..?

**Comment by loppefaaret** [comment]
The wallet from the same 24 word mnemonic key, should be identical - but the wallet database is seperate from the node blockchain database - make sure the height reported under the "Wallet" tab, is on the tip

**Reply by Peppins32** [reply]
Now that i see, on my side the wallet is "not synched", and also the height is different from the other machine, why? On both machine, full node and farming are ok, the mnemonic key is the same.

**Reply by loppefaaret** [reply]
Why is the question - It needs to sync up - Either it needs more time, or there is an error happenning - Try and see if you have any recent ERROR or WARNING message from the wallet service in your `debug.log` file, in `C:\Users\<USERNAME>\.chia\mainnet\log\` folder

**Reply by Peppins32** [reply]
I see this error: ERROR Exception: <class 'concurrent.futures._base.CancelledError'>, i also see a warning for harvester "Trying to ban localhost for 10, but will not ban". This evening when it end actual plots, i'll try to delete wallet and will see what happen (after windows restart.

**Reply by Peppins32** [reply]
Update: deleting the wallet folder (after a win restart) seems to fix it, now it's syncing again (from 0 -.-).

**Comment by BrandtH22** [comment]
@Peppins32 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
